index.js

```diff
@@ -57,253 +49,11 @@ module.exports = {
-    //       req.data.persona = req.persona;
-    //     }
-
-    //     return next();
-
-    //   }
-
-    // };
-
-    // // Add the prefix for the given persona name to the given URL.
-    // // In the presence of workflow, the persona "prefix" falls between
-    // // the workflow prefix (already in the URL) and the rest of the URL,
-    // // and varies based on locale.
-    // //
-    // // If the URL already has a persona prefix it is replaced with
-    // // one appropriate to the given persona name.
-
-    // self.addPrefix = function (req, persona, url) {
-
-    //   var workflow = self.apos.modules['apostrophe-workflow'];
-    //   var personas = self.apos.modules['apostrophe-personas'];
-    //   var liveLocale = workflow && workflow.liveify(req.locale);
-    //   var workflowPrefix = (liveLocale && workflow.prefixes && workflow.prefixes[liveLocale]) || '';
-    //   if ((require('url').parse(url).pathname || '').substr(0, workflowPrefix.length) !== workflowPrefix) {
-    //     // Workflow prefix is not actually present, probably a route like /login
-    //     workflowPrefix = '';
-    //   }
-    //   var personaInfo = (persona === 'none') ? 'none' : _.find(personas.personas, { name: persona });
-    //   var prefix;
-    //   if (personaInfo === 'none') {
-    //     prefix = '';
-    //   } else {
-    //     prefix = workflow ? personaInfo.prefixes[liveLocale] : personaInfo.prefix;
-    //   }
-    //   if (url.match(/^(https?:)?\/\//)) {
-    //     // Turn on the "slashes denote host" option
-    //     var parsed = require('url').parse(url, false, true);
-    //     parsed.pathname = prepend(parsed.pathname);
-    //     return require('url').format(parsed);
-    //   } else {
-    //     var result = prepend(url);
-    //     return result;
-    //   }
-    //   function prepend(path) {
-    //     // Watch out for null paths in edge cases when parsing the URL
-    //     path = path || '';
-    //     path = path.substr(workflowPrefix.length);
-
-    //     var existingPersonaPrefix = _.find(self.getAllPrefixes(req), function (prefix) {
-    //       return path.substr(0, prefix.length + 1) === (prefix + '/');
-    //     });
-
-    //     if (existingPersonaPrefix) {
-    //       path = path.substr(existingPersonaPrefix.length);
-    //     }
-
-    //     if (path.length) {
-    //       return workflowPrefix + prefix + path;
-    //     } else {
-    //       return workflowPrefix + prefix + '/';
-    //     }
-    //   }
-
-    // };
-
-    // self.getAllPrefixes = function (req) {
-    //   var workflow = self.apos.modules['apostrophe-workflow'];
-    //   var prefixes = [];
-    //   _.each(self.personas, function (persona) {
-    //     if (workflow) {
-    //       prefixes.push(persona.prefixes[workflow.liveify(req.locale)]);
-    //     } else if (persona.prefix) {
-    //       prefixes.push(persona.prefix);
-    //     }
-    //   });
-    //   return prefixes;
-    // };
+    self.addHelpers({
```

Comment:
Do we need this in a helper? Isn't the complete list of experiences something that only comes up when multiple-selecting them as an editor?